tests/integration/targets/grafana_organization_user/tasks/main.yml

```diff
@@ -0,0 +1,79 @@
+---
+- name: Create test user
+  grafana_user:
+    url: "{{ grafana_url }}"
+    url_username: "{{ grafana_username }}"
+    url_password: "{{ grafana_password }}"
+    name: orgtest
+    email: orgtest@example.com
+    login: orgtest
+    password: userpassword
+    state: present
+
+- name: Remove user from organization
+  community.grafana.grafana_organization_user:
+    url: "{{ grafana_url }}"
+    url_username: "{{ grafana_username }}"
+    url_password: "{{ grafana_password }}"
+    login: orgtest
+    state: absent
+  register: result
+- assert:
+    that:
+      - "result.failed == false"
+      - "result.changed == true"
```

Comment:
Can you also assert that the org and role returned are the expected one ? That will require that you add the user info in the module output.